Source/ChocolateyGui/Properties/Resources.resx

```diff
@@ -642,4 +642,7 @@ NOTE: Probably only necessary to change in RTL languages.</comment>
   <data name="PackageView_Version" xml:space="preserve">
     <value>Version</value>
   </data>
+  <data name="SettingsView_SearchInAllRepos" xml:space="preserve">
+    <value>Search in all Repos</value>
```

Comment:
Depending on the outcome of the above, we will likely want to change this text as well.